cvat-cli/src/cvat_cli/cli.py

```diff
@@ -80,11 +79,11 @@ def tasks_frames(
         Download the requested frame numbers for a task and save images as
         task_<ID>_frame_<FRAME>.jpg.
         """
-        self.client.retrieve_task(task_id=task_id).download_frames(
+        self.client.tasks.retrieve(obj_id=task_id).download_frames(
```

Comment:
I don't know if it is possible or not, but it will be great if the first argument is ID. Thus we can just write tasks.retrieve(task_id).

Reply:
I don't think I got the question. It is ID. Did you mean rename id to `id`?